src/main/scala/ai/privado/javascript/passes/methodfullname/MethodFullNameFromIdentifier.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * This file is part of Privado OSS.
+ *
+ * Privado is an open source static code analysis tool to discover data flows in the code.
+ * Copyright (C) 2022 Privado, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * For more information, contact support@privado.ai
+ *
+ */
+
+package ai.privado.javascript.passes.methodfullname
+
+import io.shiftleft.codepropertygraph.generated.nodes.Call.PropertyNames
+import io.shiftleft.codepropertygraph.generated.{Cpg, Operators}
+import io.shiftleft.codepropertygraph.generated.nodes.{Call, Expression}
+import io.shiftleft.passes.ConcurrentWriterCpgPass
+import io.shiftleft.semanticcpg.language._
+import overflowdb.traversal.Traversal
+
+import scala.jdk.CollectionConverters.CollectionHasAsScala
+
+class MethodFullNameFromIdentifier(cpg: Cpg) extends ConcurrentWriterCpgPass[(Expression, Expression)](cpg) {
+
+  val ANY_VALUE = "ANY"
+
+  /** We are solving the following scenario
+    *
+    * const log4js = require("log4js");
+    *
+    * log4js.configure({ appenders: { cheese: { type: "file", filename: "cheese.log" } }, categories: { default: {
+    * appenders: ["cheese"], level: "error" } }, });
+    *
+    * const logger = log4js.getLogger("cheese");
+    *
+    * logger.debug("Information is available);
+    *
+    * We are looking to add methodFullName to `debug`
+    *
+    * In this case the methodFullName will be pkg.log4js.getLogger.debug
```

Comment:
@khemrajrathore I guess then we can also take care of the following scenario that we were discussing in the afternoon.

```
    const users = require("../controllers/accounts.controller.js");
  
    var router = require("express").Router();
```

Reply:
ya we can try, the query for this will be slight different